examples/word-count/src/word_count.clj

```diff
@@ -13,34 +9,33 @@
             [clojure.java.io :as io]
             [clojure.tools.logging :refer [info]]
             [jackdaw.streams :as j]
-            [jackdaw.serdes.edn :as jse])
-  (:import [org.apache.kafka.common.serialization Serdes]))
-
-
-;;; Topic Configuration
-;;;
-
-(defn topic-config
-  "Takes a topic name and (optionally) a key and value serde and
-  returns a topic configuration map, which may be used to create a
-  topic or produce/consume records."
-  ([topic-name]
-   (topic-config topic-name (jse/serde) (jse/serde)))
+            [jackdaw.serdes.edn :as jse]
+            [jackdaw.serdes.resolver :as resolver]))
 
-  ([topic-name key-serde value-serde]
-   {:topic-name topic-name
+(def +topic-metadata+
+  {"input"
+   {:topic-name "input"
     :partition-count 1
     :replication-factor 1
-    :key-serde key-serde
-    :value-serde value-serde}))
+    :key-serde {:serde-keyword :jackdaw.serdes.edn/serde}
+    :value-serde {:serde-keyword :jackdaw.serdes.edn/serde}}
 
-(defn topic-names
-  []
-  ["input" "output"])
+   "output"
```

Comment:
Would be cool if these were keywords. Then it could be re-used by the test-machine code.